extensions/sql-database-projects/src/models/tree/fileFolderTreeItem.ts

```diff
@@ -75,7 +92,6 @@ function fsPathToProjectUri(fileSystemUri: vscode.Uri, projectNode: ProjectRootT
 		localUri = fileSystemUri.fsPath.substring(projBaseDir.length);
 	}
 	else {
-		vscode.window.showErrorMessage('Project pointing to file outside of directory');
```

Comment:
was this removed on purpose?

Reply:
Yup, whatever's calling this should be handling displaying errors to the user.